client/layout/index.jsx

```diff
@@ -39,16 +37,11 @@ import { isHappychatOpen } from 'state/ui/happychat/selectors';
 import SitePreview from 'blocks/site-preview';
 import { getCurrentLayoutFocus } from 'state/ui/layout-focus/selectors';
 import DocumentHead from 'components/data/document-head';
-import NpsSurveyNotice from 'layout/nps-survey-notice';
```

Comment:
It's a general issue with how dynamic imports work :) Now it's also a part of language since they proposed 'import()'.